EMOBON00092/manual_example_ro-crate-metadata.json

```diff
@@ -1,9 +1,14 @@
-{ "@context": "https://w3id.org/ro/crate/1.1/context", 
+{ "@context": {
+    "@base":"https://github.com/emo-bon/",
+    "@vocab":"https://w3id.org/ro/crate/1.2-DRAFT/context",
+    "checksum": "http://spdx.org/rdf/terms#checksum",

```

Comment:
tip:  consider raising this use case within the ro-crate community --> https://github.com/ResearchObject/ro-crate/issues

would be nice to learn how other are doint this + maybe conclude/land it as an addition into the specification?